src/azure-cli/azure/cli/command_modules/profile/__init__.py

```diff
@@ -57,6 +57,7 @@ def load_arguments(self, command):
             c.argument('use_device_code', action='store_true',
                        help="Use CLI's old authentication flow based on device code. CLI will also use this if it can't launch a browser in your behalf, e.g. in remote SSH or Cloud Shell")
             c.argument('use_cert_sn_issuer', action='store_true', help='used with a service principal configured with Subject Name and Issuer Authentication in order to support automatic certificate rolls')
+            c.argument('scopes', options_list=['--scope'], nargs='+', help='Use in the /authorize request. It can cover only one static resource.')
```

Comment:
```suggestion
            c.argument('scopes', options_list=['--scope'], nargs='+', help='Used in the /authorize request. It can cover only one static resource.')
```